js/src/redux/selectors/fallbacks.js

```diff
@@ -57,4 +57,7 @@ export const getImageFallback = state => {
  *
  * @returns {string} The authorName
  */
-export const getSiteUrl = state => ( state.analysisData.snippet.url || "" ).split( "/" )[ 0 ];
+export const getSiteUrl = () => {
+	const url = new URL( window.wpseoScriptData.metabox.base_url );
```

Comment:
@omarreiss I think it would be worth it to make it more defensive. Are there (valid) situations where window.wpseoScriptData.metabox.base_url could be empty or where the base_url would not exist? 

If so:

```
import { get } from "lodash";

const url = new URL( get( window, "wpseoScriptData.metabox.base_url", "" ) );
```

(or less of a oneliner, if you prefer)

Reply:
Thanks. Committed your suggestion, and will make it even more defensive.

Reply:
See https://github.com/Yoast/wordpress-seo/pull/16933/commits/566a4d18a463666f3d49e9ab5acbea0a11aba52e